userdocs/profiles.dev/docs/installer-docs/installing-via-gitops.md

```diff
@@ -22,6 +22,28 @@ The full setup docs can be found [here](/docs/tutorial-basics/setup#prerequisite
 
 ## Simple install from a profile URL
 
+
+### Bootstrapping your local git repository
+You can use the `flux bootstrap` command to save commonly used `pctl` configuration to your GitOps repo.
+
+Once such piece of configuration is your repo's `GitRepository` resource.
+If you have **not** bootstrapped your local GitOps repository, you will have to provide the `--git-repository` flag when installing profiles (see [below](#the-git-repository-flag) for more detail). The `--git-repository` references the namespace and name of the
+[Flux `GitRepository`](https://fluxcd.io/docs/components/source/gitrepositories/)
+resource that is pointing at your GitOps repository. The value should be in the format `<namespace>/<name>`, for example
+`flux-system/gitops-repo`. This value is needed in order for pctl to generate Flux resources, such as `Kustomization`s.
+To bootstrap your local git repository run the following:
+
+```bash
+pctl bootstrap --git-repository flux-system/gitops-repo ~/workspace/gitops-repo/
```

Comment:
This deviates from `add` as in, it accepts an absolute path instead of a relative one. Not a problem, just wanted to say that. :D

Reply:
It accepts both I believe, I just gave absolute as an example. `.` is also valid